vcpkg/reference/installation-tree-layout.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Installation tree layout
+description: This article describes the layout conventions used by vcpkg when copying build output into the installation directory.
+author: vicroms
+ms.author: viromer
+ms.date: 9/21/2023
+---
+# Installation tree layout
+
+This article describes the layout conventions used by vcpkg for the packages' installation directory. After each
+package installation, vcpkg copies the produced output into a specific location depending on the
+file's type.
+
+In classic mode, the installation directory is located in `$VCPKG_ROOT/installed` (replace
+`$VCPKG_ROOT` with your vcpkg installation path). In manifest mode, each manifest file has a
+corresponding `vcpkg_installed` directory. Regardless of the operation mode, the layout of the
+installation directory remains the same.
+
+The installation directory is created the first time a package is installed, if you don't see a
+installation directory try installing some packages first.
```

Comment:
This could use more links to other articles, such as the command line reference for the install root.